www/pages/beta.tsx

```diff
@@ -995,7 +995,7 @@ const Beta = () => {
       <Container>
         <div className="sticky inset-0 z-50">
           <div className="shadow-lg py-5 px-5 xl:px-20 bg-dark-800 flex items-center justify-between">
-            <a href="/">
+            <a id="brandLink" href="/">
```

Comment:
Good catch here. 

The actual change should be to wrap the `a` tag in a Nextjs `Link`: https://nextjs.org/docs/api-reference/next/link


Also, could you add an `a` tag to this one:

https://github.com/supabase/supabase/blob/ca957a5b4955cdcb4201940d8eb5eb194b27d02b/www/components/Nav/index.tsx#L197

That will fix the hover state across all of the pages